app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityController.scala

```diff
@@ -832,7 +832,8 @@ class YearToDateLiabilityController @Inject() (
                 draftReturn: DraftSingleDisposalReturn
               ) =>
             handleNonCalculatedEstimatedDetailsSubmit(
-              calculatedAnswers,
+              if (fillingOutReturn.isFurtherReturn.contains(true)) calculatedAnswers.unset(_.hasEstimatedDetails)
```

Comment:
you shouldn't unset `hasEstimatedDetails` here - this endpoint is capturing that field